docs/EXST bootloader.md

```diff
@@ -0,0 +1,107 @@
+# EXST bootloader
+
+The EXST (External Storage) system requires that the targets' CPU has a small program, called a bootloader, that provides functionality to load firmware from an external storage into the CPU's RAM so that the firmware can be executed.
+
+Example external storage systems include:
+* External Flash using Firmware Partition
+* File on an SD card.
+
+EXST targets also have the requirement of providing a mechansim to load and save configuration ('eeprom').  options for this include:
+
+* CPU Embedded flash page.
+* External Flash using Config Partition
+* File on an SD card.
+
+Since external storage systems can become corrupted, and communication errors can occur, Bootloaders should verify correct loading of the firmware from an external storage system, or after transferring via a debugger.  An hash is provided for this purpose.
```

Comment:
"An hash"? Any particular type of hash?

"A hash is provided for this purpose."

Reply:
keep reading the document :D MD5 currently.